lib/src/code_field/span_builder.dart

```diff
@@ -53,10 +54,10 @@ class SpanBuilder {
   TextSpan _buildNode({
     required Node node,
     required CodeThemeData? theme,
-    TextStyle? parentStyle,
+    TextStyle? ancestorStyle,
   }) {
-    final style = theme?.styles[node.className];
-    final paledStyle = _paleIfRequired(style, parentStyle);
+    final classStyle = theme?.styles[node.className];
+    final paledStyle = _paleIfRequired(classStyle ?? ancestorStyle);
```

Comment:
`classStyle ?? ancestorStyle` is duplicated below.
It's better to have `final style = theme?.styles[node.className] ?? ancestorStyle`

Reply:
fixed